Dsmr mbus support (#30052)
home-assistant/home-assistant.github.io: source/_integrations/dsmr.markdown

- Ziv E0058 ESMR5
- EasyMeter Q3D

> ### MBUS Support

Reviewer: ```suggestion
### M-Bus support
```
That's how it is spelled on the M-Bus website: https://m-bus.com/. Please correct me if I got this wrong.

> A smart meter can have multiple 'sub' devices aka mbus devices.

Reviewer: ```suggestion
A smart meter can have multiple subdevices, also known as M-Bus devices.
```

> A smart meter can have multiple subdevices, also known as M-Bus devices.

Reviewer: ```suggestion
A smart meter can have multiple subdevices, also known as [M-Bus](https://m-bus.com/) devices.
```

> For 5B the support was added for water and gas mbus devices. This means you can have up to 4 subdevices on your DSMR meter.

Reviewer: ```suggestion
For the <abbr title="Dutch smart meter requirement">DSMR</abbr> version 5B, support was added for water and gas M-Bus devices. This means you can have up to 4 subdevices on your <abbr title="Dutch smart meter requirement">DSMR</abbr> meter.
```

> While previously only 1 gasmeter was supported, there is now support for (multiple) gas and water meters.

Reviewer: ```suggestion
While previously only 1 gas meter was supported, there is now support for multiple gas and water meters.
```
